scoring.js

```diff
@@ -0,0 +1,33 @@
+function calculateScore(player) {
+  switch (player.position) {
+    case 'QB':
+      return (player.stats.rushing.yards / 10 +
+        player.stats.passing.yards / 25 +
+        player.stats.passing.touchdowns * 6 +
+        player.stats.rushing.touchdowns * 6)
+
+    case 'RB':
+      return (player.stats.receiving.receptions +
+        player.stats.rushing.yards / 10 +
+        player.stats.receiving.yards / 10 +
+        player.stats.rushing.touchdowns * 6)
```

Comment:
This return is missing some values. This doesnt work if you change the values for the test